vpr/src/base/read_options.cpp

```diff
@@ -1627,6 +1636,18 @@ argparse::ArgumentParser create_arg_parser(const std::string& prog_name, t_optio
         .help("Writes out new floorplanning constraints based on current placement to the specified XML file.")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    file_grp.add_argument(args.write_constraints_file, "--write_fix_clusters")
+        .help(
+            "Output file containing fixed locations of legalized input clusters.")
```

Comment:
Is this usable only with an input flat placement?  If so, good to mention that in the .help.